### PR TITLE
Feat: githubaction.IsHealthy() implement

### PR DESCRIPTION
**Pull request by daniel-hutao**
Signed-off-by: Daniel Hu <tao.hu@merico.dev>

# Summary

githubaction.IsHealthy() implement

### CODE REVIEW

**Comment by daniel-hutao** [comment]
Blocking #131

File: internal/pkg/util/slicez/slice.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
+package slicez
+
+// SliceInSliceInterface is used to filter the items in s1 but not in s2
+// Please use the SliceInSliceStr() or SliceInSliceInt() instead, they are more clear and easy to use.
+func SliceInSliceInterface(s1, s2 interface{}) interface{} {
+	slice1 := s1.([]interface{})
+	slice2 := s2.([]interface{})
+	retSlice := make([]interface{}, 0)
+	for _, x := range slice1 {
```
Comment (IronCore864) [inline internal/pkg/util/slicez/slice.go:9]: This is `O(n^2)`. We can do a "set difference" to make it `O(n)`.

```pseudo
result = set(s1) - set(s2)
```

**Comment by daniel-hutao** [comment]
image

**Review by IronCore864 — APPROVED** [review]

**Review by lfbdev — APPROVED** [review]
/lgtm

**Pull request merged**